SU2_CFD/src/output_structure.cpp

```diff
@@ -13389,6 +13514,30 @@ void COutput::LoadLocalData_IncFlow(CConfig *config, CGeometry *geometry, CSolve
       Variable_Names.push_back("Density");
     }
     
+    if ((Kind_Solver == NAVIER_STOKES) || (Kind_Solver == RANS)) {
+      nVar_Par += 2;
+      Variable_Names.push_back("Vorticity_x");
+      Variable_Names.push_back("Vorticity_y");
+        nVar_Par += 1; Variable_Names.push_back("Vorticity_z");
+      
+      nVar_Par +=1;
+      Variable_Names.push_back("Q_Criterion");
+    }
+
+    if ((config->GetTaylorGreen()) && (geometry->GetnDim() == 2)) {
+      
+      nVar_Par += 3;
+      Variable_Names.push_back("tgv_vel_x");
+      Variable_Names.push_back("tgv_vel_y");
+      Variable_Names.push_back("tgv_pressure");
+      
+      nVar_Par += 3;
+      Variable_Names.push_back("error_vel_x");
+      Variable_Names.push_back("error_vel_y");
+      Variable_Names.push_back("error_pressure");
+      
+    }
+    
     if (wrt_cp) {
       nVar_Par += 1;
       Variable_Names.push_back("Specific_Heat");
```

Comment:
This push_back of cp and thermal conductivity should move up 25 lines below the variable_density conditional in order to match with the order below where the values are written.